charts/dapr/charts/dapr_sidecar_injector/values.yaml

```diff
@@ -6,6 +6,11 @@ logLevel: info
 image:
   name: daprd
 
+# Specify full docker image name including registry url to use a custom injector service image
+# Otherwise, helm chart will use {{ .Values.global.registry }}/dapr:{{ .Values.global.tag }}
+injectorImage:
+  name: dapr
```

Comment:
nit: Can this be simply "" since the condition is looking for the "/" substring? Leaving it as "dapr" gives the impression that "dapr" is the value used while the actual value is the string concatenation described in the comment above.

Reply:
Sure, easily done. This was just copying the existing pattern of use in the various service deployment charts, I've gone ahead and also updated the placement, operator and sentry values.yaml for consistency.